doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -472,15 +472,19 @@ Enabling and disabling notations
    :n:`@string`
       Notations to enable or disable. :n:`@string` can be a single
       token in the notation such as "`->`" or a pattern that matches
-      the notation. See :ref:`locating-notations`.
+      the notation. See :ref:`locating-notations`. If no :n:`{? :=
+      @one_term }` is given, the variables of the notation can be
+      replaced by :n:`_`.
 
-   :n:`@qualid`
-      :ref:`Abbreviation <Abbreviations>` to enable or disable.
+   :n:`@qualid {* @ident__parm }`
+      :ref:`Abbreviation <Abbreviations>` to enable or disable, where :n:`{* @ident__parm }` are the parameters of the abbreviation. See :cmd:`Notation (abbreviation)`.
```

Comment:
```suggestion
      :ref:`Abbreviation <Abbreviations>` to enable or disable, where :n:`{* @ident__parm }`
      are the parameters of the abbreviation.
```

The two links point to the same spot (within a couple lines), so don't need both.

Reply:
Fixed and rephrased.

I also updated the error messages which had become wrong after the last change.